index.html

```diff
@@ -5,46 +5,52 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>CC Globale Summit 2015</title>
-    <link rel="stylesheet" href="main.css" />
+    <title>IT.LIVE iraq 2019</title>
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/index.css" />
 </head>
 
 <body>
-    <header class="header flex">
-        <div class="header__nav hidden">
-            <a href="#"><img src="images/brands/airbnb.svg" alt="facebook" /></a>
-            <a href="#"><img src="images/brands/airbnb.svg" alt="twitter" /></a>
-            <div class="button-group">
+    <header class="header column">
+        <div class="header__nav flex bg-dark">
+            <a href="#"><img src="images/icons/facebook.svg" alt="facebook" /></a>
+            <a href="#"><img src="images/icons/twitter.svg" alt="twitter" /></a>
+            <div class="button-group flex">
+                <button type="button" class="btn btn-text">English</button>
                 <button type="button" class="btn btn-text">My Page</button>
                 <button type="button" class="btn btn-text">Logout</button>
             </div>
         </div>
-        <a class="hidden" href="index.html"><img src="" alt="cc global summit 2015 logo"></a>
-        <button type="button" class="btn btn-icon" data-toggle="collapse"><i class="icon-menu"></i></button>
-        <nav class="hidden">
-            <ul>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
+        <nav class="nav__main flex">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
+                    class="icon-close"></i></button>
+            <ul id="mobileNav" class="nav--mobile hidden">
```

Comment:
- [ ] Please, consider fixing the mobile menu: there is an issue when the mobile menu is open and while switching to the desktop version, there is the `x` (close icon) still displaying - I know, you can easily fix it 👍 
![2022-03-13 00 15 14](https://user-images.githubusercontent.com/49064106/158033581-a33c4a90-76a8-4fb8-9831-a797faa1c8bb.gif)

> 
<img width="1052" alt="Screen Shot 2022-03-13 at 00 15 29" src="https://user-images.githubusercontent.com/49064106/158033595-4608e11a-efcb-4810-9ae6-afc19b53c20f.png">